apps/userdashboard/assets/js/a4_candy_userdashboard/api.js

```diff
@@ -0,0 +1,30 @@
+import Cookies from 'js-cookie'
```

Comment:
@Rineee @fuzzylogic2000 @phillimorland I spent quite a bit of time thinking about that. but I think it is beneficial to extract that from the components. it makes them pretty messy if we leave it in there, especially if we have to add more different requests. so the best way i found, was to - again - put it in its own file, what do you think about and is there a more elegant way? I mean better would be to implement it in a4 but it's most probably not the right time?